app/javascript/controllers/tooltips_controller.js

```diff
@@ -0,0 +1,10 @@
+import { Controller } from '@hotwired/stimulus'
+import * as bootstrap from 'bootstrap' // eslint-disable-line no-unused-vars
```

Comment:
Are you sure this is necessary? (It might be, but might already be imported elsewhere.)

Reply:
Sadly, it is necessary here.